mne/report.py

```diff
@@ -1414,6 +1456,33 @@ def _render_cov(self, cov_fname, info_fname):
                                          show=show)
         return html
 
+    def _render_whitened_evoked(self, evoked_fname, noise_cov):
+        """Show whitened evoked.
+        """
+        global_id = self._get_id()
+
+        evokeds = read_evokeds(evoked_fname, verbose=False)
+
+        html = []
+        for ev in evokeds:
+            noise_cov = regularize(noise_cov, ev.info)
+            ev_white = whiten_evoked(ev, noise_cov, diag=True)
+            global_id = self._get_id()
+
```

Comment:
I think if the method is unknown, it can safely assume that it is `empirical`, at least for the purposes of plotting. And that probably needs `read_evokeds(..., baseline=(None, 0))` to fix the baseline.